api/config/test.exs

```diff
@@ -0,0 +1,28 @@
+import Config
+
+# Configure your database
+#
+# The MIX_TEST_PARTITION environment variable can be used
+# to provide built-in test partitioning in CI environment.
+# Run `mix help test` for more information.
+config :api, Api.Repo,
+  username: "postgres",
+  password: "postgres",
+  hostname: "localhost",
+  database: "performance_manager",
+  port: "5432",
```

Comment:
connect to the DB created by docker-compose for testing.

the assumption is that we're running some scripts using the py package first and then accessing the db it modifies to run tests.

potentially all of this gets orchestrated via docker-compose at some point.